guides/common/modules/proc_installing-the-snapshot-management-plugin.adoc

```diff
@@ -0,0 +1,12 @@
+[id="Installing_the_Snapshot_Management_Plugin_{context}"]
+= Installing the Snapshot Management Plug-in
+
+Perform the following step to install the _Snapshot Management_ plug-in on your {Project}.
```

Comment:
I find that line 4 and line 7 are very repetitive, but in general I am fine with this.